src/renderer/components/WatchAccountButton.tsx

```diff
@@ -45,6 +71,7 @@ function WatchAcountPopover(props: {
             <Col sm={{ span: 10, offset: 2 }}>
               <Button
                 type="button"
+                disabled={!!validationError || !toKey}
```

Comment:
sorry to be a pain, but you should be able to take the `!!` out now, due to the use of `undefined`